test/metabase/dashboard_subscription_test.clj

```diff
@@ -18,9 +19,9 @@
     :or   {channel :email}}
    f]
   (mt/with-temp* [Pulse         [{pulse-id :id, :as pulse}
-                                 (-> pulse
-                                     (merge {:name         "Aviary KPIs"
-                                             :dashboard_id (u/the-id dashboard)}))]
+                                 (->> pulse
```

Comment:
Changed this for consistency, so that the `pulse` argument can override the hard-coded details